tests/api_workflow/mocked_api_workflow_client.py

```diff
@@ -1075,7 +1036,6 @@ class MockedApiWorkflowClient(ApiWorkflowClient):
     n_embedding_rows_on_server = N_FILES_ON_SERVER
 
     def __init__(self, *args, **kwargs):
-        lightly.api.api_workflow_client.ApiClient = MockedApiClient
```

Comment:
This line was overwriting the `ApiClient` globally (for all tests), so that it was raising an error if you accidentally called it.
The global overwrite caused the new unittest to fail, thus I had to remove it. It was not needed anyway and only helped you when writing new unittests, as it improved the error message.

Reply:
Now that we write new unittests differently, we don't need it anymore.